<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Error_Log</class>
 <widget class="QWidget" name="Error_Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>155</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current_time">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>310</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sick Log</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1280</width>
     <height>660</height>
    </rect>
   </property>
   <widget class="QLabel" name="line_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>85</y>
      <width>960</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>145</y>
      <width>960</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>205</y>
      <width>960</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>265</y>
      <width>960</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>325</y>
      <width>960</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>385</y>
      <width>960</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>445</y>
      <width>960</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>505</y>
      <width>960</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="line_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>565</y>
      <width>960</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Errtime_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Err time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Errcode_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Err code:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pgm_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pgm:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Sta_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sta:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Step_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Cyc_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>160</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cyc:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Link_label">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>160</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Link:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Freetime_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Free time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Hilimit_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hilimit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Iolimit_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lolimit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Tpv_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tpv=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Hpv_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hpv=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Wpv_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wpv=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Tsv_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tsv=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Hsv_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hsv=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Wsv_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wsv=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Out_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="Temp_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Temp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Humi_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Humi:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Svr_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>520</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Svr:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Page_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>590</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Page=</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Footer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>720</y>
     <width>1280</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="font_page_pbtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saving_pbtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_log_pbtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ClearErrLog</string>
    </property>
   </widget>
   <widget class="QLabel" name="footer_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>3</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
